tools/docgen/docgen/manifest.yaml

```diff
@@ -1,4 +1,35 @@
 channels:
-  - name: "channels.WebsiteChannel"
-    input_dir: "../../../docs/"
-    output_dir: "../../../website/docs/"
+#  - name: channels.WebsiteChannel
+#    input_dir: ../../../docs/
+#    output_dir: ../../../website/docs/
+  - name: channels.FabricChannel
+    input_dir: ../../../docs/
+    output_dir: ../../../target/fabric-docs-pr/
+    notebooks:
+      - path: Explore Algorithms/Responsible AI/Foo.ipynb
+        metadata:
+          title: Causal Structure
+          description: Causal Structure
+          ms.topic: overview
+          ms.custom: build-2023
+          ms.reviewer: jessiwang
+          author: JessicaXYWang
+          ms.author: jessiwang
+      - path:  Explore Algorithms/AI Services/Multivariate Anomaly Detection.ipynb
+        metadata:
+          title: Analyze time series
+          description: Use SynapseML and Azure Cognitive Services for multivariate anomaly detection.
+          ms.topic: overview
+          ms.custom: build-2023
+          ms.reviewer: jessiwang
+          author: JessicaXYWang
+          ms.author: jessiwang
+      - path: Explore Algorithms/Causal Inference/Quickstart - Measure Causal Effects.ipynb
```

Comment:
Add rest of notebooks and make sure it matches what you had in your previous branch